.travis.yml

```diff
@@ -7,14 +7,6 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
-    - env: TOXENV=py27
-      python: 2.7
-    - env: TOXENV=py27-pinned
-      python: 2.7
-    - env: TOXENV=py27-extra-deps
-      python: 2.7
-    - env: TOXENV=pypy
-      python: 2.7
```

Comment:
In this file you also need to update the deployment code below, which is currently tied to the `py27`.

Reply:
@Gallaecio indeed, I was going to ask what to change there

Reply:
@kmike You probably have though about this a few times.

One thing I can think of is renaming `py37` to `py`, keep `py` always for the latest version (i.e. when Python 3.8 is release, `py` becomes 3.8 and `py37` is created), and use `py` in the deployment code.